docs/source/conf.py

```diff
@@ -71,7 +72,7 @@
 # General information about the project.
 project = "Kornia"
 author = f"{project} developers"
-copyright = f"2019, {author}"
+copyright = f"{datetime.date.today().year}, {author}"
```

Comment:
https://docs.astral.sh/ruff/rules/call-date-today/

Reply:
Looks like it's fixed

Reply:
awesome